app/services/mark_patient_mobile_numbers.rb

```diff
@@ -0,0 +1,26 @@
+class MarkPatientMobileNumbers
+  def self.call
+    new.call
+  end
+
+  def call
+    return unless ENV["FORCE_MARK_PATIENT_MOBILE_NUMBERS"] == "true"
```

Comment:
Do you want to use Flipper for this?

Reply:
I thought about it. I think I like the idea of long-running switches like this in Flipper. You know what, I'll do it 👍

Reply:
I was hoping, that we eventually migrate all our feature toggles or flags of all kind to flipper, keep bare-minimum config in the config.

Reply:
I was on the fence, but I think you've sold me. Done in 5156590